subcommand/server-acl-init/command.go

```diff
@@ -491,6 +497,60 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGateways) > 0 {
+		// Create a token for each ingress gateway name
```

Comment:
Can you comment as to *why* we need to create a token for *each* gateway even though they all have the same policy (because after the fact users may need to attach different policies to each gateway token depending on what the gateway's config is)